app/src/main/java/com/android/sample/model/recipe/RecipesViewModel.kt

```diff
@@ -99,6 +99,9 @@ class RecipesViewModel(
     viewModelScope.launch {
       getCategoryList()
       fetchRandomRecipes(NUMBER_RECIPES_TO_FETCH)
+      getAllDownloads(
+          onSuccess = { _ -> Log.d(LOG_TAG_RECIPE_VIEWMODEL, "Downloaded recipes fetched") },
```

Comment:
Small nitpick, but could you make the error message a constant

Reply:
Done!